src/pages/workspace/WorkspaceMembersPage.tsx

```diff
@@ -339,8 +301,20 @@ function WorkspaceMembersPage({policyMembers, personalDetails, route, policy, se
                 }
             }
 
+            const isOwner = policy?.owner === details.login;
             const isAdmin = session?.email === details.login || policyMember.role === CONST.POLICY.ROLE.ADMIN;
 
+            let roleBadge = null;
+            if (isOwner || isAdmin) {
+                roleBadge = (
```

Comment:
The badge wasn't being styled properly when removing admin while offline in the native apps: https://github.com/Expensify/App/issues/37776